app/src/main/java/ch/epfl/sweng/SDP/Constants.java

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.sweng.SDP;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+public class Constants {
```

Comment:
The class itself is not so useful, we could just embed the constants inside Account.java as fields

Reply:
It is useful. It's a so-called wrapper that allows me to test Account with mockito.

Reply:
Ah my bad!